<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsoleTab</class>
 <widget class="QWidget" name="ConsoleTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1164</width>
    <height>276</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="opaqueResize">
      <bool>true</bool>
     </property>
     <property name="handleWidth">
      <number>0</number>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="RenameTabFrame" name="renameTabFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="ConnectionBar" name="connectionBar">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
     </widget>
     <widget class="ConsoleView" name="consoleView">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ContiguousSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="resizeMode">
       <enum>QListView::Adjust</enum>
      </property>
      <property name="selectionRectVisible">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="LogfilePanel" name="logPanel">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="HighlightingsFrame" name="highlightingsFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="StatusBarFrame" name="statusBar">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionChangeColor">
   <property name="text">
    <string>Change Color...</string>
   </property>
  </action>
  <action name="actionConnection">
   <property name="text">
    <string>&amp;Connection...</string>
   </property>
   <property name="toolTip">
    <string>Connection</string>
   </property>
  </action>
  <action name="actionChangeFont">
   <property name="text">
    <string>Change Font...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About superterm</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionHighlight">
   <property name="text">
    <string>Highlight...</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Fullscreen</string>
   </property>
  </action>
  <action name="actionLogging">
   <property name="text">
    <string>Logging...</string>
   </property>
  </action>
  <action name="actionToggleAutoscroll">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Autoscroll</string>
   </property>
  </action>
  <action name="actionChangeFontColor">
   <property name="text">
    <string>Change Font Color...</string>
   </property>
   <property name="toolTip">
    <string>Change Font Color</string>
   </property>
  </action>
  <action name="actionRenameTab">
   <property name="text">
    <string>&amp;Rename tab</string>
   </property>
   <property name="toolTip">
    <string>Rename tab</string>
   </property>
  </action>
  <action name="actionToggleTimeStamps">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Timestamps</string>
   </property>
  </action>
  <action name="actionAlignHorizontal">
   <property name="text">
    <string>Align horizontally</string>
   </property>
  </action>
  <action name="actionAlignVertical">
   <property name="text">
    <string>Align vertically</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StatusBarFrame</class>
   <extends>QFrame</extends>
   <header>src/ui/statusbarframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HighlightingsFrame</class>
   <extends>QFrame</extends>
   <header>src/ui/highlightingsframe.h</header>
   <container>1</container>
   <slots>
    <signal>highlightingChanged()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>LogfilePanel</class>
   <extends>QFrame</extends>
   <header>src/ui/logfilepanel.h</header>
   <container>1</container>
   <slots>
    <signal>loggingStarted()</signal>
    <signal>loggingStopped()</signal>
    <slot>onConfigurationChanged(QString)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>ConnectionBar</class>
   <extends>QFrame</extends>
   <header>src/ui/connectionbar.h</header>
   <container>1</container>
   <slots>
    <signal>connectClicked()</signal>
    <signal>saveClicked()</signal>
    <signal>configurationChanged(QString)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>RenameTabFrame</class>
   <extends>QFrame</extends>
   <header>src/ui/renametabframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConsoleView</class>
   <extends>QListView</extends>
   <header>src/ui/consoleview/consoleview.h</header>
   <slots>
    <signal>keyPressed(QKeyEvent*)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionChangeColor</sender>
   <signal>triggered()</signal>
   <receiver>ConsoleTab</receiver>
   <slot>showColorDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionConnection</sender>
   <signal>triggered()</signal>
   <receiver>connectionBar</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionChangeFont</sender>
   <signal>triggered()</signal>
   <receiver>ConsoleTab</receiver>
   <slot>showFontDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClear</sender>
   <signal>triggered()</signal>
   <receiver>ConsoleTab</receiver>
   <slot>clearTab()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHighlight</sender>
   <signal>triggered()</signal>
   <receiver>highlightingsFrame</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>ConsoleTab</receiver>
   <slot>showAboutDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFullscreen</sender>
   <signal>triggered()</signal>
   <receiver>ConsoleTab</receiver>
   <slot>toggleFullScreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLogging</sender>
   <signal>triggered()</signal>
   <receiver>ConsoleTab</receiver>
   <slot>showLogToFileFrame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>logPanel</sender>
   <signal>loggingStarted()</signal>
   <receiver>ConsoleTab</receiver>
   <slot>startLogging()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>logPanel</sender>
   <signal>loggingStopped()</signal>
   <receiver>ConsoleTab</receiver>
   <slot>stopLogging()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionToggleAutoscroll</sender>
   <signal>triggered()</signal>
   <receiver>ConsoleTab</receiver>
   <slot>toggleAutoScroll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>connectionBar</sender>
   <signal>connectClicked()</signal>
   <receiver>ConsoleTab</receiver>
   <slot>onConnectClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>581</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>connectionBar</sender>
   <signal>saveClicked()</signal>
   <receiver>ConsoleTab</receiver>
   <slot>showSaveDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>581</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>connectionBar</sender>
   <signal>configurationChanged(QString)</signal>
   <receiver>ConsoleTab</receiver>
   <slot>onConfigurationChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>581</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionChangeFontColor</sender>
   <signal>triggered()</signal>
   <receiver>ConsoleTab</receiver>
   <slot>showFontColorDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRenameTab</sender>
   <signal>triggered()</signal>
   <receiver>ConsoleTab</receiver>
   <slot>showRenameTabDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionToggleTimeStamps</sender>
   <signal>triggered()</signal>
   <receiver>ConsoleTab</receiver>
   <slot>toggleTimeStamps()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>consoleView</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>ConsoleTab</receiver>
   <slot>showContextMenu(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>581</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAlignHorizontal</sender>
   <signal>triggered()</signal>
   <receiver>ConsoleTab</receiver>
   <slot>alignWindowsHorizontally()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onConnectClicked()</slot>
  <slot>onComboChanged()</slot>
  <slot>onContextMenuRequested(QPoint)</slot>
  <slot>showContextMenu(QPoint)</slot>
  <slot>showColorDialog()</slot>
  <slot>showConnectBar()</slot>
  <slot>showFontDialog()</slot>
  <slot>updateHighlightings()</slot>
  <slot>showAboutDialog()</slot>
  <slot>toggleFullScreen()</slot>
  <slot>startLogging()</slot>
  <slot>stopLogging()</slot>
  <slot>toggleAutoScroll()</slot>
  <slot>onKeyPressed(QKeyEvent*)</slot>
  <slot>onMoreClicked()</slot>
  <slot>showSaveDialog()</slot>
  <slot>hideConnectBar()</slot>
  <slot>onConfigurationChanged(QString)</slot>
  <slot>showFontColorDialog()</slot>
  <slot>showRenameTabDialog()</slot>
  <slot>clearTab()</slot>
  <slot>toggleTimeStamps()</slot>
  <slot>showLogToFileFrame()</slot>
  <slot>alignWindowsHorizontally()</slot>
 </slots>
</ui>
